docs/api/browser-window.md

```diff
@@ -265,6 +265,8 @@ It creates a new `BrowserWindow` with native properties as set by the `options`.
     * `offscreen` Boolean - Whether to enable offscreen rendering for the browser
       window. Defaults to `false`.
     * `sandbox` Boolean - Whether to enable Chromium OS-level sandbox.
+    * `vibrancy` String - Add a type of vibrancy effect to the window, only on
```

Comment:
> See supported types at the `setVibrancy` method.

It might seem silly but can we duplicate the list of value up here as well purely so the JSON api is accurate